test/integration/render/scenarios/geometries/line-string/width/scenario.js

```diff
@@ -1,10 +1,14 @@
-const map = new carto.Map({
+const map = new mapboxgl.Map({
```

Comment:
What are the reasons for this change?

Reply:
I just moved a test in mgl-integration here and replace the previous one as part or `geometry tests` reorder. This cover lines with MGL.